### DISCUSSION TITLE
2022년 5월 3일(화) 저녁 번개

### DISCUSSION THREAD

**Discussion by wooyaggo86** [ThinkAboutSoftware/AcademicConference#208] [General]
~~안녕하세요 김우현 입니다. 제가 잠시 한국에 귀국해 있는데요. 
5월 3일 저녁에 서울에서 시간 되시는 분들 모셔서 직접 뵙고 같이 이야기 나누고 싶습니다 :)
아직 장소는 정하지 않았지만, 가능한 분들 위치를 고려해 편리한 곳을 결정하려고 합니다.
많은 분들 뵙고 싶은데, 가능하시다면 꼭 알려주세요🙏~~

개인 사정으로 인해 진행이 어렵게 됬습니다. 죄송합니다..

**Comment by jongfeel** [comment]
지금쯤 오셔서 한국에 계실거라 생각은 했는데 이번 금요일 모임때 얘기해 보려고 했거든요.

먼저 시간을 내실 수 있는 날을 알려주셔서, 저는 우선 시간을 비울 수 있습니다!

---

그리고 년도를 2022년으로 수정했습니다!

**Reply by wooyaggo86** [reply]
악 감사합니다😂

**Reply by TaeHyoungKwon** [reply]
저도 참가하겠습니다~

**Reply by Joseph-Cha** [reply]
강남 쪽에서 만나면 퇴근 후 바로 참석이 가능할 것 같아요 ㅎㅎ
특별한 이슈가 없으면 칼퇴가 가능할 것 같지만 상황이 어떻게 될지 내일 봐야할 것 같습니다!ㅠㅠ

**Comment by wooyaggo86** [comment]
개인적으로 안좋은일이 있어서 조금 황망하네요.. 
내일 일정을 다 취소하고 광주에 좀 더 있어야할 것 같습니다.
죄송합니다...

**Reply by jongfeel** [reply]
알겠습니다.

그래도 돌아가기 전에 뵐 수 있는 기회가 있으면 좋겠다는 생각이 드네요!
